### ISSUE TITLE
message with map<int64, int64> field isn't properly handled (is(create()) returns false)

### ISSUE THREAD

**Issue by kskalski** [timostamm/protobuf-ts#260]
For message

```proto
message Foo {
    map<int64, int64> longs_map = 1;
}
```

the following code
```ts
const foo = Foo.create({ longsMap: { '1': 2n } });
console.log('Foo is foo', Foo.is(foo));
```
prints
```
Foo is foo false
```

The same happens for different long type option, e.g. with ``long_type_number`` and ``Foo.create({ longsMap: { '1': 2 } })``

**Comment by kskalski** [comment]
I suppose the problem is with value spec for ``map<int64, int64>`` field, which doesn't contain the ``L`` type. For the message
```proto
int64 long_field = 2;
map<int64, int64> longs_map = 1;
```
the spec looks like this:
```ts
{ no: 2, name: "long_field", kind: "scalar", T: 3 /*ScalarType.INT64*/, L: 2 /*LongType.NUMBER*/ },
{ no: 1, name: "longs_map", kind: "map", K: 3 /*ScalarType.INT64*/, V: { kind: "scalar", T: 3 /*ScalarType.INT64*/ } }
```

and ``V`` part of the ``map`` field spec only has ``T: 3`` without ``L``

**Comment by timostamm** [comment]
Good catch! The interpreter adds it correctly, but it isn't generated. Fixed in https://github.com/timostamm/protobuf-ts/commit/58e1b791a46cfd25af775482fdb926e9dc7d936c.

**Comment by timostamm** [comment]
Fix released in [v2.4.0](https://github.com/timostamm/protobuf-ts/releases/tag/v2.4.0).
